crypto/src/lib.rs

```diff
@@ -164,15 +164,16 @@ impl KeyGenConfiguration {
     }
 }
 
-/// Pair of Public and Private keys.
-#[cfg_attr(feature = "ffi", derive(IntoFfi, TryFromFfi))]
-#[derive(Debug, Clone, PartialEq, Eq, Getters, Serialize)]
-#[getset(get = "pub")]
-pub struct KeyPair {
-    /// Public Key.
-    public_key: PublicKey,
-    /// Private Key.
-    private_key: PrivateKey,
+ffi! {
```

Comment:
idea is that this macro will replace the structure it wraps with
```rs
pub struct KeyPair {
    ptr: [u8; 0]
}
```

in the wasm smartcontract